script/build-flynn

```diff
@@ -0,0 +1,226 @@
+#!/bin/bash
+
+set -e
+
+ROOT="$(cd "$(dirname "$0")/.." && pwd)"
+source "${ROOT}/script/lib/ui.sh"
+
+usage() {
+  cat <<USAGE >&2
+usage: $0 [options]
+
+Build Flynn using the builder image.
+
+OPTIONS:
+  -h, --help              Show this message
+  -v, --verbose           Be verbose
+  -x, --version=VERSION   Explicit version to use [default: dev]
+  -t, --tuf-keys=KEYS     TUF root keys
+  --git-version           Generate the version using git status
+USAGE
+}
+
+main() {
+  local version="dev"
+  local verbose=false
+  local tuf_keys=""
+
+  while true; do
+    case "$1" in
+      -h | --help)
+        usage
+        exit 0
+        ;;
+      -v | --verbose)
+        verbose=true
+        shift
+        ;;
+      -x | --version)
+        if [[ -z "$2" ]]; then
+          fail "--version flag requires an argument"
+        fi
+        version="$2"
+        shift 2
+        ;;
+      -t | --tuf-keys)
+        if [[ -z "$2" ]]; then
+          fail "--tuf-keys flag requires an argument"
+        fi
+        tuf_keys="$2"
+        shift 2
+        ;;
+      --git-version)
+        version="$(git_version)"
+        shift
+        ;;
+      *)
+        break
+        ;;
+    esac
+  done
+
+  if [[ $# -ne 0 ]]; then
+    usage
+    exit 1
+  fi
+
+  # TODO: remove
+  rm -f "${ROOT}/cli/tuf.go"
+
+  local flynn_host="${ROOT}/build/bin/flynn-host"
+
+  # if building from clean, download binaries + images
+  if ! [[ -e "${flynn_host}" ]]; then
+    local sha="132903ae94d7c1b174261e738424b94022233d938ef091659727a30bddeb7a61d114b762f0fc2ffeecb967aef87d92bc71a79b9c3df31220c24ee9d8d83fa932"
+    local url="https://s3.amazonaws.com/flynn-temp/tuf/targets/${sha}.flynn-host.gz"
```

Comment:
I was imagining releasing this as a branch when we feel it is ready to merge then updating the references just before merging.

This may also be a chance to do a clear up of the main TUF repo, and perhaps change the key structure, WDYT?